legacy/testdata/rulesets.json

```diff
@@ -1175,6 +1175,118 @@
             }
         }
     },
+    {
```

Comment:
just testing existing migration of date rules - we can add to this when we figure out what should populated in the `_ui` section